Tests/SKCoreTests/ToolchainRegistryTests.swift

```diff
@@ -560,16 +560,25 @@ private func makeToolchain(
     makeExec(binPath.appending(component: "swiftc\(execExt)"))
   }
 
-  let dylibExt = Platform.currentPlatform?.dynamicLibraryExtension ?? ".so"
+  let dylibSuffix = Platform.currentPlatform?.dynamicLibraryExtension ?? ".so"
 
   if sourcekitd {
     try! fs.createDirectory(libPath.appending(component: "sourcekitd.framework"))
     try! fs.writeFileContents(libPath.appending(components: "sourcekitd.framework", "sourcekitd") , bytes: "")
   }
   if sourcekitdInProc {
-    try! fs.writeFileContents(libPath.appending(component: "libsourcekitdInProc\(dylibExt)") , bytes: "")
+#if os(Windows)
+    try! fs.writeFileContents(binPath.appending(component: "sourcekitdInProc\(dylibSuffix)") , bytes: "")
```

Comment:
This matches how Toolchain.swift works, but  I wonder if we should instead have
* libPath = binPath on Windows
* dylibPrefix = "lib" **except** on Windows

I think then the only weird case is that libIndexStore has "lib" even on Windows.

WDYT?

Reply:
I don't think that we want "libPath = binPath" on Windows.  libpath is lib - that is where the import libraries are.  binpath is bin - that is where the dll is.  Having a dylibPrefix makes sense, but it seemed that we would only have a single use given the oddity of `libIndexStore.dll`.

Reply:
Okay fair enough